how-to/kubeflow_assets.md

```diff
@@ -0,0 +1,164 @@
+# Kubeflow Assets
+
+This doc is intended to provide an overview of the different assets 
+which are used by the Kubeflow community and identify the respective owners.
+
+## GitHub Org: kubeflow
+
+https://github.com/kubeflow
+
+The Kubeflow GitHub org is administered declaratively.
+
+Admins can be found in [org.yaml](https://github.com/kubeflow/internal-acls/blob/e4303ff3c7299bde05b4a9c7519e8592c5137755/github-orgs/kubeflow/org.yaml#L7)
+
+More info about how the GitHub org is administered in [GitHub Admin](https://github.com/kubeflow/community/blob/master/how-to/github_admin.md)
+
+
+### GitHub Robots
+
+https://github.com/kubeflow-bot
+
+* Password is stored in GCP Secret Manager project `kubeflow-admin`
+* You will also need a recovery token which are also stored in secret manager
+
+  * These are onetime use so remove it when you use it
+  * When you login you need to select use recovery token.
+
+## kubeflow.org GSuite Domain
+
+Admins
```

Comment:
Believe this is immutable? Is anyone collecting overall github statistics for the org?

Reply:
What do you mean by GitHub statistics? Policy doesn't allow us for granting non-googlers admin privileges. Also following the principle of least privilege we don't want to grow the number of root users/admins.
In accordance with that, we are trying to great policy groups (https://github.com/kubeflow/internal-acls) that have privileges needed to fulfill different roles (e.g. test/release admin, drive manager etc...) and then add folks as necessary to those groups.

Ideally everything is automated through GitOps so there is very little need to have root privileges.